config/components/registries-conf/registries_conf_configmap.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: e2e-registries-conf
+  namespace: olmv1-system
```

```suggestion
  namespace: system
```
The namespace is overwritten by the value in `kustomization.yaml`?